src/Data/Functor/Costar.purs

```diff
@@ -0,0 +1,49 @@
+module Data.Functor.Costar where
+
+import Prelude
+
+import Control.Comonad (class Comonad, extract)
+import Control.Extend (class Extend, (=<=))
+
+import Data.Distributive (class Distributive, distribute)
+import Data.Functor.Invariant (class Invariant, imapF)
+import Data.Functor.FunctorRight (class FunctorRight)
+import Data.Newtype (class Newtype)
+
+-- | `Costar` turns a `Functor` into a `Profunctor` "backwards".
```

Comment:
This could mention that it also turns a `Contravariant` into a `Bifunctor`... but maybe that doesn't need to be said explicitly? I'm more on the fence with this one than with my other doc suggestions.